src/lib/components/TestSlotWrapper.svelte

```diff
@@ -4,6 +4,7 @@
   import type { ComponentType, SvelteComponentTyped } from "svelte";
 
   export let Component: ComponentType<SvelteComponentTyped>;
+  export let slotContent: string | Node;
```

Comment:
Unsure if `Node` is the right alternative here or not, but I figure we can update this as needed later.